src/js/content_scripts/input-mapper.js

```diff
@@ -80,6 +88,30 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
         });
     };
 
-    // Create an instance of the component.
-    gamepad.inputMapper();
+    // Create an instance of the inputMapper when a new page is opened.
+    gamepad.inputMapperInstance = null;
+    gamepad.inputMapperInstance = gamepad.inputMapper();
+
+    /**
+     * Restore or destroy the instance of the inputMapper when the visibility of current
+     * window/tab is changed.
+     */
+    document.addEventListener("visibilitychange", function () {
```

Comment:
So, I don't understand why this doesn't address #34.  I mean, if you're managing to destroy the input mapper, how are the other windows able to take action on inputs?

Reply:
This handles most of the parts when the window is minimized, or the tab is changed (though we don't have problems in tabs). But when you open a new window, the old window is still running (as what I observed) and this [Visibility API](https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API) doesn't account for that. So, those issues are handled by it, but on minimizing the window.